src/Template/Element/Form/related_translations.twig

```diff
@@ -17,7 +17,8 @@
             <div v-show="isOpen" class="tab-container">
 
                 <div>
-                    {{ Property.control('lang', object.attributes.lang, { 'label': __('The main language is') })|raw }}
+                    {% set lang = object.id ? object.attributes.lang : config('I18n.default')|default('en') %}
```

Comment:
We should use `Project.config.I18n.default` here and not `I18n.default`

See `App\Form\Options::lang()`

`I18n` config key refers to the Manager UI languages and `Project.config.I18n` refers to the object languages - the two sets can be different. 

Anyway I would remove `|default('en')` here
It should be:
```
{% set lang = object.id ? object.attributes.lang : config('Project.config.I18n.default') %}
```